eth/filters/test_backend.go

```diff
@@ -0,0 +1,173 @@
+package filters
```

Comment:
is it a generated file? if so where the comments at the beginning of the file?

Reply:
No, it is not a generated file.